### PR TITLE
whale 4

### PR DESCRIPTION
**Pull request by fsansalvadore**

### CODE REVIEW

File: apps/www/_blog/2024-08-15-log-drains.mdx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  - platform
+tags:
+  - launch-week
+  - o11y
+  - logging
+date: '2024-08-15'
+toc_depth: 3
+---
+
+Today, Supabase is releasing Log Drains for all Team and Enterprise users in Public Alpha.
```
Comment (kiwicopple) [inline apps/www/_blog/2024-08-15-log-drains.mdx:20]:
```suggestion
Today, Supabase is releasing Log Drains for all Team and Enterprise users.
```

I think we should put " in Public Alpha" at the end under a "Getting started" section or something. These blog posts live on the internet forever, and "Public Alpha" is a snapshot in time

File: apps/www/_blog/2024-08-15-log-drains.mdx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+With Log Drains, developers can now export logs generated by their Supabase products to external destinations, such as Datadog or custom HTTP endpoints. Logs generated by all Supabase products such as the Database, Storage, Realtime and Auth are supported.
```
Comment (kiwicopple) [inline apps/www/_blog/2024-08-15-log-drains.mdx:22]:
```suggestion
With Log Drains, developers can export logs generated by their Supabase products to external destinations, such as Datadog or custom HTTP endpoints. All logs generated by Supabase products such as the Database, Storage, Realtime and Auth are supported.
```

File: apps/www/_blog/2024-08-15-log-drains.mdx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+Beyond providing a single pane of glass inside your existing logging and monitoring system, Log Drains can be used to build additional alerting and observability pipelines. For example, you can ingest Postgres connection logs into your Security Information and Event Management (SIEM) or Intrusion Detection System (IDS) to create custom alerting rules based on events happening in your database.
+
+This feature also allows for extended retention periods to meet compliance requirements.
+
+While we continue to improve logging and alerting within the Supabase platform, Log Drains provide an important escape hatch for these advanced use cases today.
```
Comment (kiwicopple) [inline apps/www/_blog/2024-08-15-log-drains.mdx:28]:
```suggestion
This feature also allows for extended retention periods to meet compliance requirements and provide an important escape hatch for advanced use cases while we continue to improve logging and alerting within the Supabase platform
```

File: apps/www/_blog/2024-08-15-log-drains.mdx [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+The initial supported destinations are:
+
+- Datadog Logs
+- HTTP Endpoint
+
+Popular destinations like Datadog are supported out of the box. More detailed setup guides are available within the [log drains documentation](https://supabase.com/docs/guides/platform/log-drains).
+
+<Img alt="Datadog Logs drain" src="/images/blog/lw12/day-4/datadog-example.png" />
+
+For the providers that are not supported out of the box, the HTTP Endpoint drain can be used to send logs to any destination that supports ingestion via HTTP POST requests. For example, you can send logs to an Edge Function, filter, or restructure the logs, and then dispatch them to an external provider. In the following example, we perform a simple `console.log` of the received JSON payload. Detailed setup guide is available under the [Edge Functions guide](https://supabase.com/docs/guides/functions/log-drains).
```
Comment (kiwicopple) [inline apps/www/_blog/2024-08-15-log-drains.mdx:45]:
```suggestion
For the providers that are not natively supported yet, the HTTP Endpoint drain can be used to send logs to any destination that supports ingestion via HTTP POST requests. For example, you can send logs to an Edge Function, filter, or restructure the logs, and then dispatch them to an external provider. In the following example, we perform a simple `console.log` of the received JSON payload. Detailed setup guide is available under the [Edge Functions guide](https://supabase.com/docs/guides/functions/log-drains).
```

File: apps/www/_blog/2024-08-15-log-drains.mdx [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+<Img
+  alt="HTTP Endpoint drain to Supabase Edge Functions"
+  src="/images/blog/lw12/day-4/edge-function-example.png"
+/>
+
+When self-hosting or during local development, Log Drains will be available through the Studio and can be under Project Settings > Log Drains.
```
Comment (kiwicopple) [inline apps/www/_blog/2024-08-15-log-drains.mdx:52]:
```suggestion
Log Drains are available for self-hosting and local development through the Studio under Project Settings > Log Drains.
```

File: apps/www/_blog/2024-08-15-log-drains.mdx [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+## The Supabase Analytics server
+
+Log Drains is built on [Logflare](https://github.com/Logflare/logflare), the analytics and observability server of the Supabase stack.
```
Comment (kiwicopple) [inline apps/www/_blog/2024-08-15-log-drains.mdx:56]:
```suggestion
Log Drains are built into [Logflare](https://github.com/Logflare/logflare), the analytics and observability server of the Supabase stack.
```

**Review by craigcannon — APPROVED** [review]

File: apps/www/_blog/2024-08-15-log-drains.mdx (hunk 5 above)
Comment (Ziinc) [reply apps/www/_blog/2024-08-15-log-drains.mdx]: PR not yet released as its blocked

**Pull request merged**